zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskController.java

```diff
@@ -121,4 +128,38 @@ private CompletableFuture<ResponseEntity<Object>> updateUserTask(
                 .withAuthentication(RequestMapper.getAuthentication())
                 .updateUserTask(request.userTaskKey(), request.changeset(), request.action()));
   }
+
+  @PostMapping(
+      path = "/user-tasks/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<Object> searchUserTasks(
+      @RequestBody(required = false) final UserTaskSearchQueryRequest query) {
+    return SearchQueryRequestMapper.toUserTaskQuery(
+            query == null ? new UserTaskSearchQueryRequest() : query)
+        .fold(this::search, RestErrorMapper::mapProblemToResponse);
+  }
+
+  private ResponseEntity<Object> search(final UserTaskQuery query) {
+    try {
+      final var tenantIds = TenantAttributeHolder.tenantIds();
+      final var result =
+          userTaskServices.withAuthentication((a) -> a.tenants(tenantIds)).search(query);
```

Comment:
❌ You can use `RequestMapper.getAuthentication()` and put this into the `withAuthentication` method. That should add everything you need.
```suggestion
      final var result =
          userTaskServices.withAuthentication(RequestMapper.getAuthentication()).search(query);
```